.palantir/revapi.yml

```diff
@@ -66,6 +66,12 @@ acceptedBreaks:
       old: "method void org.apache.iceberg.io.DataWriter<T>::add(T)"
       justification: "Removing deprecated method"
   "1.1.0":
+    org.apache.iceberg:iceberg-api:
```

Comment:
I think this should land under `1.2.0` and not `1.1.0` because the change is being introduced after 1.2.0 was released (most likely because the branch was created before 1.2.0 was released). @karuppayya could you please follow-up with that and move this under 1.2.0?

Reply:
Ah yeah I forgot we just missed the release

Reply:
I've opened https://github.com/apache/iceberg/pull/7223 to address that. @RussellSpitzer could you review please?